### ISSUE TITLE
Context Improperly Passed Downstream on start

### ISSUE THREAD

**Issue by abellgithub** [LASzip/LASzip#74]
If the first point has a context (scan channel) other than 0 (say, 2) and the second point also has that same channel, encoders other than the first (POINT14) will be passed the wrong context. This is because `context`, the argument passed to `LASwriteItemCompressed_POINT14_v3::write` is only set on change. Since there is no change, `channel` is never set. For example, when encoding points of type PRDF 7, the function `LASwriteItemCompressed_RGB14_v3::write` will be passed 0, rather than 2.

**Comment by rapidlasso** [comment]
I vaguely remember that exactly was the reason that there is now a v4 ... or is the same error there also?

**Comment by abellgithub** [comment]
V4 is different. Is that what's supposed to be used? When I use `laszip_create()` it uses V3. I remember that you told me V4 wasn't used. Has that changed?

**Comment by rapidlasso** [comment]
Yes, V4 is supposed to be used eventually. Maybe now is the time to flip the switch?
